settings.gradle.kts

```diff
@@ -68,7 +68,7 @@ dependencyResolutionManagement {
             version("log4j-version", "2.17.2")
             version("netty-version", "4.1.66.Final")
             version("protobuf-java-version", "3.19.4")
-            version("swirlds-version", "0.31.0-alpha.1")
+            version("swirlds-version", "0.32.0-alpha.0")
```

Comment:
I was told it's `0.32.0-alpha.1` (with a 1 at the end)?

Reply:
Ah yes. Thanks ! Fixed it.